### ISSUE TITLE
Character `[ ` unusable in text, keymap fr_BE, because of Keybinding

### ISSUE THREAD

**Issue by Krands** [atom/atom#12771]
### Prerequisites

* [x] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

### Description

Hi, in Atom-core, when I write text (such as pseudo code or markdown), I can't have access to the `[` character, because the keybinding is defined as such : 

| Keystroke | Command | Source | Selector |
|----------|-------------|--------|-------|
|ctrl-[ | editor:outdent-selected-rows | Core | atom-workspace atom-text-editor:not([mini]) |

Here is the _Belgian Keyboard Layout_ 
![belgian keyboard layout](http://www.keysourcechina.com/img/img/img/20117122320473499.jpg)

As you can see, the `[ ` character in on level 3 on Belgian layout, where on English, it's on the first level. As we need `ctrl-alt` (or `alt-gr`) to write ´[´, nothing can happen because of the keybinding in the same time. This happens on Windows and it might happen on Linux too. I am not sure on macOS / OS X, however.

### Steps to Reproduce

1. Change keyboard layout to Belgian (`fr_BE`)
2. Try to use `[` in ATOM

**Expected behavior:** Get `[`.

**Actual behavior:** Nothing happens.

### Versions

```
Atom: 1.10.2
Electron: 0.37.8
Chrome: 49.0.2623.75
Node: 5.10.0
``` 
On Windows 10.

Greetings !

**Comment by Ben3eeE** [comment]
Duplicate of https://github.com/atom/atom-keymap/issues/35.

**Comment by 50Wliu** [comment]
Will be fixed starting with Atom 1.12.0.

**Comment by Krands** [comment]
Ho yes, sorry. I did not make a search within closed issues, nor took more time for the good repo.

Lesson learned.
Thank you!
